examples/with-docker/.eslintrc.json

```diff
@@ -1,3 +1,4 @@
 {
+  "root": true,
```

Comment:
What does this do?

Reply:
> The `"root"` field in an ESLint configuration file is a boolean that tells ESLint whether the configuration file is the root or not.
>
> When ESLint runs, it starts from the directory where it was called and searches upwards for configuration files. However, once it finds a configuration file with `"root": true`, it stops looking for other configuration files.
>
> Here's what it does:
>
> - If `"root": true` is in a configuration file, ESLint won't search for any other configuration files in parent folders.
> - If `"root": false` or the `"root"` key is not present, ESLint will continue searching upwards through the directory structure looking for more configuration files.
>
> This can be useful in monorepo setups or when you have different ESLint configurations for different parts of your project. By setting `"root": true` in a configuration file, you can ensure that ESLint uses only that configuration for the directory and its subdirectories.

each of those eslintrc files are the root eslints of their respective project. Without this field you get some errors with the up to date ESLint and ESLint typescript pkgs